test/Spec.hs

```diff
@@ -57,7 +56,7 @@ primopExample =
   in App (Annot (Primop ConcatString) (inferPrimop ConcatString))
          (pair (StrV "abc") (StrV "xyz"))
 
--- > case (0, "string") of
+-- > unpack (0, "string") of
```

Comment:
fwiw I like the syntax `unpack (x, y) = rhs in body` or simply `let (x, y) = rhs in body`